test/unit/math/fwd/mat/fun/mdivide_right_tri_test.cpp

```diff
@@ -1,6 +1,20 @@
 #include <stan/math/fwd/mat.hpp>
 #include <gtest/gtest.h>
 
+TEST(AgradFwdMatrixMdivideRightTri, simple) {
+  using Eigen::Lower;
+  using Eigen::MatrixXd;
+  using Eigen::RowVectorXd;
+  using stan::math::mdivide_right_tri;
+
+  MatrixXd b(1, 2);
+  b << 2, 3;
+  MatrixXd A(2, 2);
+  A << 3, 0, 1, 4;
+  MatrixXd b_under_A = mdivide_right_tri<Lower>(b, A);
+  SUCCEED();
```

Comment:
What's the purpose of this test? It looks like it's just checking that it doesn't crash. If so, look at the death test macros.